src/Orleans.CodeGenerator/Model/ProxyInterfaceDescription.cs

```diff
@@ -81,7 +81,12 @@ private List<ProxyMethodDescription> GetMethods(INamedTypeSymbol symbol)
             {
                 foreach (var method in iface.GetDeclaredInstanceMembers<IMethodSymbol>())
                 {
-                    if (methods.TryGetValue(method, out _))
+                    if (method.MethodKind == MethodKind.ExplicitInterfaceImplementation)
```

Comment:
I think this should also check for a body, explicit impls can be used for reabstraction in interface type hierarchies.

Reply:
I am not sure it's necessary: the generated proxy should implement every method explicitly (I added a comment below this line). Any overrides should take effect on the callee side naturally (correct me if you find that I'm wrong here)